camp/camp-brooklyn/src/main/java/org/apache/brooklyn/camp/brooklyn/spi/creation/BrooklynComponentTemplateResolver.java

```diff
@@ -257,6 +257,7 @@ public boolean canResolve() {
         new BrooklynEntityDecorationResolver.EnricherSpecResolver(yamlLoader).decorate(spec, attrs, encounteredRegisteredTypeIds);
         new BrooklynEntityDecorationResolver.InitializerResolver(yamlLoader).decorate(spec, attrs, encounteredRegisteredTypeIds);
         new BrooklynEntityDecorationResolver.SpecParameterResolver(yamlLoader).decorate(spec, attrs, encounteredRegisteredTypeIds);
+        new BrooklynEntityDecorationResolver.TagsResolver(yamlLoader).decorate(spec, attrs, encounteredRegisteredTypeIds);
```

Comment:
@tbouron is onto something here. For example for effectors we go through an initializer and try not to expand the keywords too much.

I think that anything that is a "first class concept" on an `EntitySpec` should have the corresponding way to express in yaml. There are tags on specs, so `brooklyn.tags` in CAMP makes sense. Similarly for `brooklyn.parameters`, `brooklyn.config`. There are no effectors on the `EntitySpec` so there are no `brooklyn.effectors` in CAMP.